introduction.md

```diff
@@ -1,3 +1,3 @@
 ~My name is Danny
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?